src/bp/migrations/v12_15_1-1607952538-add_thumbs_down_misunderstood.ts

```diff
@@ -0,0 +1,29 @@
+import * as sdk from 'botpress/sdk'
+import { Migration, MigrationOpts } from 'core/services/migration'
+
+const migration: Migration = {
```

Comment:
can you change migration target (in file name ) to 12_16_0 please

Reply:
Will do